### PR TITLE
docs: add maintainers area, communicate project goals

### PR DESCRIPTION
**Pull request by jakebolam**
Provide some documentation for project maintainers.

Set goals for the next few months.

Track weekly metrics on project health.

Open to feedback on all of this!

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Metric Targets for April 2019
+Date | Maintainers | All Contributors | Stars | Bot Installs | Bot Stars | Bot Messages | Bot PRs | Website Sessions | Seo Hits
+---|---|---|---|---|---|---|---|---|---
+Wed April 30th | 4 | 40 | 2700 | 250 | 100 | 50 | 25 | 500 | 10
+
+
+## Metrics OKR Period - Open Leaders (Mid Feb 2019 until April 2019)
+Date | Maintainers | All Contributors | Stars | Bot Installs | Bot Stars | Bot Messages | Bot PRs | Website Sessions | Seo Hits
+---|---|---|---|---|---|---|---|---|---
+Wed Feb 20th | 4 | 34 | 2442 | 107 | 75 | 21 | 20 | 382 | 0
```
Comment (jakebolam) [inline MAINTAINERS.md]: Thoughts on other ways of doing this?

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+---|---|---|---|---|---|---
+**TARGET for April 201** | 4 | 40 | 2700 | 10 | 500 | 10
+Wed Feb 20th | 4 | 34 | 2442 | 7 | 382 | 0
+Wed Feb 13th | 4 | 33 | 2430 | 6 | 431 | 0
+
+
+### Bot
+Date | Bot Installs | Bot Stars | Bot Messages | Bot PRs 
+---|---|---|---|---
+**TARGET for April 2019** | 250 | 100 | 50 | 40
```
Comment (jakebolam) [inline MAINTAINERS.md]: Wanting to be able to measure our great progress!

Thinking of recording metrics, and setting targets? Thoughts?

File: MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Roadmap
+- All Contributors Bot in Marketplace (Requires 250 bot installs)
+- All Contributors Website Portal?
+
+## Metrics OKR Period - Open Leaders (Mid Feb 2019 until April 2019)
+### Project & Website
+Date | Maintainers | Contributors |Stars | Website Languages  | Website Sessions | Seo Hits
+---|---|---|---|---|---|---
+**TARGET for April 201** | 4 | 40 | 2700 | 10 | 500 | 10
```
Comment (jakebolam) [inline MAINTAINERS.md]: Wanting to be able to measure our great progress!

Thinking of recording metrics, and setting targets? Thoughts?

Comment (Berkmann18) [reply MAINTAINERS.md]: That's a good idea.
How often would those metrics and targets be recorded and checked?

File: MAINTAINERS.md (hunk 2 above)
Comment (Berkmann18) [reply MAINTAINERS.md]: I'm cool with that, I suppose Sentry offers a way to get those metrics onto a file or something which could then be used for this document.

File: MAINTAINERS.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+Wed Feb 20th | 4 | 34 | 2442 | 7 | 382 | 0
+Wed Feb 13th | 4 | 33 | 2430 | 6 | 431 | 0
+
+
+### Bot
+Date | Bot Installs | Bot Stars | Bot Messages | Bot PRs 
+---|---|---|---|---
+**TARGET for April 2019** | 250 | 100 | 50 | 40
+Wed Feb 20th | 107 | 75 | 21 | 20
+Wed Feb 13th | 82 | 71 | 24 | 23
```
Comment (jakebolam) [inline MAINTAINERS.md]: From @gr2m.

Should we track uninstalls of the bot?

Comment (Berkmann18) [reply MAINTAINERS.md]: It would be useful to know.

Comment (jakebolam) [reply MAINTAINERS.md]: Ticketed: https://github.com/all-contributors/all-contributors-bot/issues/148

File: MAINTAINERS.md (hunk 3 above)
Comment (jakebolam) [reply MAINTAINERS.md]: I'm planning to add them each Wednesday.

I wonder if we can track those automatically on a dashboard somewhere, any ideas?

**Comment by jakebolam** [comment]
Merging first past, open to iterate on this

File: MAINTAINERS.md (hunk 3 above)
Comment (Berkmann18) [reply MAINTAINERS.md]: > I wonder if we can track those automatically on a dashboard somewhere, any ideas?

In regards to Lighthouse, we can use a [NodeJS script](https://github.com/GoogleChrome/lighthouse/blob/master/docs/readme.md#using-programmatically) or by using the [CLI](https://github.com/GoogleChrome/lighthouse#using-the-node-cli).

Comment (jakebolam) [reply MAINTAINERS.md]: Good idea!

**Pull request merged**